addon/webextension/manifest.json

```diff
@@ -1,6 +1,19 @@
 {
-  "name": "pug-webextension",
-  "version": "1.0.4",
+  "name": "Looking Glass",
+  "applications": {
+    "gecko": {
+      "id": "pug.experience.mrrobot@shield.mozilla.org",
+      "strict_min_version": "57.0",
+      "strict_max_version": "62.0"
+    }
+  },
+  "developer": {
+    "name": "PUG Experience Group (Gregg Lind and others) <glind@mozilla.com>",
+    "url":  "https://support.mozilla.org/kb/lookingglass"
+  },
+  "description": "MY REALITY IS JUST DIFFERENT THAN YOURS.\n\nLooking Glass is a collaboration between Mozilla and the makers of Mr. Robot to provide a shared world experience. Are you a fan of Mr. Robot? If so, join the hunt for answers!\n\nParticipating in this shared world experience requires explicit user opt in. If you are not actively participating in the ARG (Augmented Reality Game) no modifications will be made to Firefox.\n\nhttps://support.mozilla.org/kb/lookingglass",
+
+  "version": "1.0.5",
```

Comment:
Do we want to add an icon for about:addons so we don't have the default puzzle piece?

<img width="1280" alt="add-ons_manager" src="https://user-images.githubusercontent.com/557895/34133975-c8b577c6-e40c-11e7-88fb-e0b4ea784ae6.png">

Reply:
I don't have an asset there.  Noted.